### PR TITLE
Issue 18780 show host name in container selector

### PR DESCRIPTION
**Pull request by freddyucv**

### CODE REVIEW

File: src/app/portlets/dot-edit-page/shared/services/dot-edit-layout.service.spec.ts [hunk 1]
```diff
@@ -17,7 +17,10 @@ describe('DotEditLayoutService', () => {
                 identifier: '5363c6c6-5ba0-4946-b7af-cf875188ac2e',
```
Comment (alfredo-dotcms) [inline src/app/portlets/dot-edit-page/shared/services/dot-edit-layout.service.spec.ts:17]: migrate to `TestBed`

File: src/app/view/components/dot-container-selector/dot-container-selector.component.spec.ts [hunk 2]
```diff
@@ -52,7 +52,10 @@ describe('ContainerSelectorComponent', () => {
                     identifier: '427c47a4-c380-439f',
```
Comment (alfredo-dotcms) [inline src/app/view/components/dot-container-selector/dot-container-selector.component.spec.ts:52]: migrate to `TestBed`

File: src/app/view/components/dot-container-selector/dot-container-selector.component.spec.ts [hunk 3]
```diff
@@ -62,12 +65,31 @@ describe('ContainerSelectorComponent', () => {
...
+                    parentPermissionable: {
+                        hostname: 'demo.dotcms.com'
+                    }
                 }
             ];
         })
     );
 
+    it('should show the hots name and container name',
+        fakeAsync(() => {
```
Comment (alfredo-dotcms) [inline src/app/view/components/dot-container-selector/dot-container-selector.component.spec.ts:78]: fakeAsync not needed

File: src/app/portlets/dot-edit-page/shared/services/dot-edit-layout.service.spec.ts (hunk 1 above)
Comment (freddyucv) [reply src/app/portlets/dot-edit-page/shared/services/dot-edit-layout.service.spec.ts:16]: done https://github.com/dotCMS/core-web/pull/1419/commits/9f0329227f49d5e2ba679ec450bf7170112f69e3#diff-9fdb7f7a5d8c1c883513d66fe700c574R93

File: src/app/view/components/dot-container-selector/dot-container-selector.component.spec.ts (hunk 2 above)
Comment (freddyucv) [reply src/app/view/components/dot-container-selector/dot-container-selector.component.spec.ts:78]: done https://github.com/dotCMS/core-web/pull/1419/commits/9f0329227f49d5e2ba679ec450bf7170112f69e3#diff-f80340d948cb93edaa053f4b5ac39829R38

File: src/app/view/components/dot-container-selector/dot-container-selector.component.spec.ts (hunk 3 above)
Comment (freddyucv) [reply src/app/view/components/dot-container-selector/dot-container-selector.component.spec.ts]: done https://github.com/dotCMS/core-web/pull/1419/commits/9f0329227f49d5e2ba679ec450bf7170112f69e3#diff-f80340d948cb93edaa053f4b5ac39829L78

**Pull request merged**